execution_engine/src/core/engine_state/mod.rs

```diff
@@ -2052,29 +2052,8 @@ where
             )
             .map_err(Into::into)?;
 
-        let next_era_validators = {
-            let mut era_validators = match self.get_era_validators(
-                correlation_id,
-                GetEraValidatorsRequest::new(post_state_hash, step_request.protocol_version),
-            ) {
-                Ok(era_validators) => era_validators,
-                Err(error) => {
-                    return Err(StepError::GetEraValidatorsError(error));
```

Comment:
Sorry for not catching this earlier, but we can get rid of this error variant now